.github/actions/package_csharp/scripts/mac_build_csharp.sh

```diff
@@ -7,11 +7,23 @@ export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=2
 echo "COREBINARYDIRECTORY: ${COREBINARYDIRECTORY}"
 echo "CTEST_SOURCE_DIRECTORY: ${CTEST_SOURCE_DIRECTORY}"
 
+# Detect OS
+OS_NAME=$(uname -s)
+
+# Detect version (for macOS)
```

Comment:
The OSX deployment is the oldest version of the OS supported. I believe for all languages this should remain 10.9 or Intel architectures and should be 11.0 for ARM. OSX 11 was the first to support the ARM architecture.